spec/example_app/db/schema.rb

```diff
@@ -122,4 +122,36 @@
   add_foreign_key "orders", "customers"
   add_foreign_key "pages", "products"
   add_foreign_key "payments", "orders"
+
+  create_view "customers_index", sql_definition: <<-SQL
+      SELECT customers.id,
+      customers.name,
+      customers.email,
+      customers.created_at,
+      customers.updated_at,
+      customers.email_subscriber,
+      customers.kind,
+      customers.country_code,
+      customers.example_time,
+      customers.password,
```

Comment:
Since this is an index-specific view, can you remove `password` and other unused fields, which likely won't be displayed on our index page?

Reply:
Oh, I see in `db/views` you're using `select customers.*`; removing `password` would require much more code and make the view brittle to changes.

Leave as-is, all good.